packages/recommender/src/recommendation-grpc.ts

```diff
@@ -42,15 +41,15 @@ export function loadRecommendationService() {
  * it serves.
  * @return The new server object
  */
-export function createGRPCRecommendationServer(port = '0.0.0.0:50051') {
+export function createGRPCRecommendationServer(port = '0.0.0.0:0') {
```

Comment:
Below it says `50000`, here it is `0`. Why the mismatch?

Reply:
We have two cases:

1. Set `0` to get an available port and use the assigned port for testing
2. Set `50000` as a fixed port so that it can be configured statically in datasource json.

Reply:
Can we use `host` instead of `port` because `host` is conventionally `hostname[:port]`. We should update in all other relevant places.

Reply:
That's the grpc term.